pkg/aws/aws_client/cloudwatch_logs.go

```diff
@@ -0,0 +1,38 @@
+package aws_client
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
+	"github.com/openshift-online/ocm-common/pkg/log"
+)
+
+func (client *AWSClient) DescribeLogGroupsByName(logGroupName string) (cloudwatchlogs.DescribeLogGroupsOutput, error) {
+	output, err := client.CloudWatchLogsClient.DescribeLogGroups(context.TODO(), &cloudwatchlogs.DescribeLogGroupsInput{
+		LogGroupNamePrefix: &logGroupName,
+	})
+	if err != nil {
+		log.LogError("Got error describe log group:%s ", err)
+	}
+	return *output, err
```

Comment:
We are assuming the output is never nil, I would suggest we either do a nil check before returning or  return the pointer and do the nil check later on. Just to avoid any nasty nil pointers